lib/presentation/view/header/context_menu_header_builder.dart

```diff
@@ -40,18 +42,22 @@ class ContextMenuHeaderBuilder {
 
   ContextMenuHeaderBuilder(this._key, this._file);
 
-  ListTile build() {
-    return ListTile(
-      key: _key,
-      leading: _file.fileIcon(),
-      title: Transform(
-        transform: Matrix4.translationValues(-16, -2, 0.0),
-        child: Text(
-          _file.fileName(),
-          maxLines: 1,
-          style: TextStyle(fontSize: 14, color: AppColor.documentNameItemTextColor),
+  Transform build() {
```

Comment:
Can you explain the different from List tile with Transform

Reply:
ListTile is used to display the list box
Transform is used to align ListTile on the header context menu, taking ListTile as a child widget